core/spec/models/spree/stock/default_shipping_rate_selector_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'spec_helper'
+
+describe Spree::Stock::ShippingRateSelector do
+  context '#sort' do
```

Comment:
I would use `#describe` when describing a public method, and then multiple `contents` inside the `#describe` if needed.